src/main/java/sg/ncl/MainController.java

```diff
@@ -230,16 +235,16 @@ public String loginSubmit(@ModelAttribute("loginForm") LoginForm loginForm, Mode
             headers.set("Authorization", "Basic " + base64Creds);
 
             HttpEntity<String> request = new HttpEntity<String>("parameters", headers);
-            ResponseEntity responseEntity = restTemplate.exchange(properties.getSioAuthUrl(), HttpMethod.POST, request, String.class);
-
-            // TODO call the proper validation functions
-            jwtTokenString = responseEntity.getBody().toString();
+            responseEntity = restTemplate.exchange(properties.getSioAuthUrl(), HttpMethod.POST, request, String.class);
         } catch (Exception e) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)